test-data/unit/check-selftype.test

```diff
@@ -108,9 +108,9 @@ reveal_type(B.new())  # E: Revealed type is '__main__.B*'
 
 [case testSelfTypeOverride]
 from typing import TypeVar, cast
- 
```

Comment:
Next time you submit a PR, can you instruct your editor *not* to clean up trailing whitespace on lines you otherwise didn't touch? While it looks better, the many trivial chunks in the diff are distracting during a review.